libraries/SRV_Channel/SRV_Channels.cpp

```diff
@@ -213,6 +213,14 @@ const AP_Param::GroupInfo SRV_Channels::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("_DSHOT_ESC",  24, SRV_Channels, dshot_esc_type, 0),
 
+    // @Param: _GPIO_MASK
+    // @DisplayName: Servo GPIO mask
+    // @Description: This sets a bitmask of outputs which will be available as GPIOs. Any auxillary output with either the function set to -1 or with the corresponding bit set in this mask will be available for use as a GPIO pin
+    // @Bitmask: 0: Servo 1, 1: Servo 2, 2: Servo 3, 3: Servo 4, 4: Servo 5, 5: Servo 6, 6: Servo 7, 7: Servo 8, 8: Servo 9, 9: Servo 10, 10: Servo 11, 11: Servo 12, 12: Servo 13, 13: Servo 14, 14: Servo 15, 15: Servo 16
```

Comment:
can we remove the spaces between the numbers and the following string?  e.g. "0:Servo 1" instead of "0: Servo 1".  You could remove the space between "Servo" and "1" as well if you like.